app/shared/filters.js

```diff
@@ -67,7 +67,7 @@ angular.module('portainer.filters', [])
 .filter('nodestatusbadge', function () {
   'use strict';
   return function (text) {
-    if (text === 'Unhealthy') {
+    if (text == 'down' || text == 'unhealthy') {

```

Comment:
One important thing to notice is that state is case-sensitive. Also, this `nodestatusbadge` filter is actually used for Docker Swarm nodes **AND** Docker Swarm mode nodes. 

Docker Swarm nodes can have the following states:

* Healthy
* Unhealthy

Docker Swarm Mode nodes can have the following states:

* ready
* down

And maybe some other states, but they are undocumented.

This should be fine if you update `(text == 'down' || text == 'unhealthy')` to `(text == 'down' || text == 'Unhealthy')`